zebrad/src/commands/start.rs

```diff
@@ -203,6 +205,17 @@ impl StartCmd {
             misbehavior_sender.clone(),
         );
 
+        // Start health server if configured
+        info!("initializing health endpoints");
+        let (health_task_handle, _health_addr) = crate::components::health::init(
```

Comment:
Nitpick: Could we add this to the list of imports from `components` at the top of this file?

```suggestion
        let (health_task_handle, _health_addr) = health::init(
```